File: ts/wam/7952.ts
export const WamFsApiSignupFlow = 7952 as const

export interface FsApiSignupFlowEvent {
  /** field 1, wire `signup_entry_point` */
  signupEntryPoint?: SIGNUPENTRYPOINT
  /** field 2, wire `signup_user_journey_operation` */
  signupUserJourneyOperation?: SIGNUPUSERJOURNEYOPERATION
  /** field 3, wire `thread_creation_time` */
  threadCreationTime?: THREADCREATIONTIME
  /** field 4, wire `thread_id_hmac` */
  threadIdHmac?: string
  /** field 5, wire `unified_session_id` */
  unifiedSessionId?: string
}

File: rust/wam/7952.rs
pub const WAM_FS_API_SIGNUP_FLOW: u32 = 7952;

#[derive(Debug, Default)]
pub struct FsApiSignupFlowEvent {
    /// field 1, wire `signup_entry_point`
    pub signup_entry_point: Option<SIGNUPENTRYPOINT>,
    /// field 2, wire `signup_user_journey_operation`
    pub signup_user_journey_operation: Option<SIGNUPUSERJOURNEYOPERATION>,
    /// field 3, wire `thread_creation_time`
    pub thread_creation_time: Option<THREADCREATIONTIME>,
    /// field 4, wire `thread_id_hmac`
    pub thread_id_hmac: Option<String>,
    /// field 5, wire `unified_session_id`
    pub unified_session_id: Option<String>,
}